pyserini/prebuilt_index_info.py

```diff
@@ -7263,4 +7263,212 @@
         "downloaded": False,
         "texts": "wiki-all-6-3-tamber"
     },
+    "miracl-v1.0-ar-mdpr-tied-pft-msmarco-ft-miracl-ar": {
+        "description": "Faiss index for MIRACL v1.0 (Arabic) corpus encoded by mDPR passage encoder pre-fine-tuned on MS MARCO, then fine-tuned in-language with MIRACL.",
+        "filename": "faiss.miracl-v1.0-ar.mdpr-tied-pft-msmarco-ft-miracl-ar.20230329.e40d4a.tar.gz",
+        "readme": "faiss.miracl-v1.0-ar-mdpr-tied-pft-msmarco-ft-miracl-ar.20230329.e40d4a.README.md",
```

Comment:
We're missing a README? We can have a single README shared by all the languages...

Reply:
Yeah we do have one, I'll correct the README name